FXyz-Importers/src/main/java/org/fxyz3d/importers/maya/values/impl/MArrayImpl.java

```diff
@@ -35,14 +35,12 @@
 import java.util.ArrayList;
 import java.util.Iterator;
 import java.util.List;
-import org.fxyz3d.importers.maya.MayaImporter;
 import org.fxyz3d.importers.maya.types.MArrayType;
 import org.fxyz3d.importers.maya.values.MArray;
 import org.fxyz3d.importers.maya.values.MData;
 
 public class MArrayImpl extends MDataImpl implements MArray {
-    public static final boolean DEBUG = MayaImporter.DEBUG;
-    public static final boolean WARN = MayaImporter.WARN;
+    public static final boolean WARN = false;
```

Comment:
Should we remove this at all?

Reply:
Sure, I'll see if there are any other places with similar patterns. I'll just replace with a Logger